docs/user/commercial/organizations.rst

```diff
@@ -3,13 +3,19 @@ Organizations
 
 .. include:: /shared/admonition-rtd-business.rst
 
-Organizations allow you to segment who has access to what projects in your company.
-Your company will be represented as an Organization,
-let's use ACME Corporation as our example.
+In this article, we explain how the *organization* feature on Read the Docs allows you to manage access to your projects.
+On Read the Docs for Business, your account is linked to an *organization*, whereby you can define both individual and common read and write access to your projects or collections of projects.
```

Comment:
```suggestion
On |com_brand|, your account is linked to an *organization*.
Organizations allow you to define both individual and team permissions to your projects.
```

We should basically never say `whereby` :)

Reply:
Ah yes, and should basically never say "Read the Docs for Business".

I couldn't think of a way to construct a sembr linter... but we could have a ban list of words in .rst, like "whereby" and "Read the Docs for Business" :)

Reply:
Yea, we use vale for this on WTD: https://github.com/writethedocs/www/tree/main/vale/WTD